DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -39,6 +49,11 @@ VerboseTestTargetFalseEnabledProtocols = Enabled Protocols for website "{0}" do
 VerboseTestTargetFalseDefaultPage = Default Page for website "{0}" does not match the desired state.
 VerboseTestTargetTrueResult = The target resource is already in the desired state. No action is required.
 VerboseTestTargetFalseResult = The target resource is not in the desired state.
+VerboseTestTargetFalsePreload = Preload for website "{0}" do not match the desired state.
+VerboseTestTargetFalseAutoStart = AutoStart for website "{0}" do not match the desired state.
+VerboseTestTargetFalseAutoStartProvider = AutoStartProvider for website "{0}" does not match the desired state.
+VerboseTestTargetFalseSSLFlags = SSLFlags are not in the desired state.
```

Comment:
There is a separate resource named **xSSLSettings**. I think `SSLFlags` can be removed for now.